specification/compiler/model/utils.ts

```diff
@@ -555,6 +551,12 @@ function hoistPropertyAnnotations (property: model.Property, jsDocs: JSDoc[]): v
       property.identifier = value
     } else if (tag === 'doc_url') {
       property.docUrl = value
+    } else if (tag === 'variant') {
+      if (value === 'container_property') {
+        property.container_property = true
+      } else {
+        throw new Error(`Unknown 'variant' value '${value}' on property ${property.name}`)
+      }
```

Comment:
Please use the internal assertion tool, which will show to the user which part of the input spec caused the error :)
```suggestion
      assert(jsDocs, value === 'container_property', `Unknown 'variant' value '${value}' on property ${property.name}`)
      property.container_property = true
```